scripts/integration-test

```diff
@@ -27,10 +28,37 @@ while ! kubectl rollout status -w -n cattle-system deploy/rancher-webhook >/dev/
     echo "Waiting for rancher to deploy rancher-webhook..."
     sleep 2
 done
+echo "Webhook deployed"
 
 # After rancher deploys webhook kill the bash command running tail.
 kill ${TAIL_PID}
 
+# Wait for helm operation to complete and save rancher-webhook release info before we kill rancher and the cluster.
+while
+    pods=$(kubectl get pods -n cattle-system --field-selector=status.phase=Running -o jsonpath='{.items[?(@.metadata.generateName=="helm-operation-")].metadata.name}')
+    [[ -n "$pods" ]]
+do
+    echo "Waiting for help operation to finish"
+    sleep 2
+done
+
+# Kill Rancher since we only need the CRDs and the intial webhook values.
+# We do not want Rancher to reconcile an older version of the webhook during test.
+kill ${RANCHER_PID}
```

Comment:
Could we instead change the `rancher-webhook-version` setting to `$TAG` to prevent it from reconciling a different value?

Reply:
I will also add that I was able to run 60 consecutive iterations of the drone pipeline check for flakiness.